contrib/sirupsen/logrus/example_test.go

```diff
@@ -0,0 +1,29 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package logrus
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func ExampleHook() {
+	//Setup logrus, do this once at the beginning of your program
```

Comment:
```suggestion
	// Setup logrus, do this once at the beginning of your program
```
Can you please add the space in other comments too?

Reply:
👍 will do. I'll also fix my goland setup / go fmt command to find and check this automatically since this isn't the first time I've accidentally forgotten a space :P